server/src/main/java/io/crate/planner/operators/LogicalPlanVisitor.java

```diff
@@ -69,6 +69,10 @@ public R visitInsert(InsertFromValues logicalPlan, C context) {
         return visitPlan(logicalPlan, context);
     }
 
+    public R visitAbstractJoinPlan(AbstractJoinPlan logicalPlan, C context) {
```

Comment:
This was needed to make it testable, to be able to operate JoinGraph on any kind of Join not just JoinPlan. This makes sense anyway.

Reply:
It's fine but I don't get how it helps to test. for me the test works without those changes.